src/seedu/addressbook/data/person/Address.java

```diff
@@ -9,10 +9,24 @@
 public class Address {
 
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must entered in the "
+            + "following format a/BLOCK, STREET, UNIT, POSTAL_CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String ADDRESS_SPLIT_REGEX = ",";
 
+    // Address is represented as an array of 4 elements.
+    private static final int NUMBER_OF_ADDRESS_ELEMENTS = 4;
+    private static final int BLOCK_INDEX = 0;
+    private static final int STREET_INDEX = 1;
+    private static final int UNIT_INDEX = 2;
+    private static final int POSTAL_CODE_INDEX = 3;
+    
     public final String value;
```

Comment:
What is the point of this variable now?

toString() method needs to be updated using the new components.